src/routes/NewUser/components/NewUserPage/NewUserPage.js

```diff
@@ -140,12 +196,20 @@ function NewUser() {
   // };
 
   async function handleFormSubmit(values) {
+    const newValues = values;
     if (!userLocationInfo) {
       alert('Please select a location above.'); // eslint-disable-line no-alert
       return;
     }
 
-    const userUpdates = { ...values, ...userLocationInfo };
+    delete userLocationInfo.lookedUpAddress;
+
+    // Default the displayName if it isn't already set (e.g., when signing up using email).
+    if (userData.get('displayName') === null) {
```

Comment:
We might want to go with `!userData.get('displayName')` to handle other falsey cases such as when we have set the displayName to an empty string